packages/react-components/react-timepicker-compat/stories/TimePicker/index.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { TimePicker } from '@fluentui/react-timepicker-compat';
+
+import descriptionMd from './TimePickerDescription.md';
+import bestPracticesMd from './TimePickerBestPractices.md';
+
+export { Default } from './TimePickerDefault.stories';
+export { Controlled } from './TimePickerControlled.stories';
+export { FreeformWithErrorHandling } from './TimePickerFreeformWithErrorHandling.stories';
+export { FreeformCustomParsing } from './TimePickerFreeformCustomParsing.stories';
+export { TimePickerWithDatePicker } from './TimePickerWithDatePicker.stories';
+
+export default {
+  title: 'Preview Components/TimePicker',
```

Comment:
```suggestion
  title: 'Compat Components/TimePicker',
```

https://github.com/microsoft/fluentui/commit/3dc2af6a1cd38696ae00d8654d751b6f160d9b50#diff-2555c540fa8d7922ca74fdd1250227ae7e6af126735fd2fb3f5c5e6f56c02ac4R248